Kernel/ApplicativeHelper.hpp

```diff
@@ -73,6 +74,18 @@ class ApplicativeHelper {
   static bool isExactApplied(TermList head, unsigned argNum);
   static bool isOverApplied(TermList head, unsigned argNum);
   static bool isSafe(TermStack& args);
+  // TODO is this right?
```

Comment:
@ibnyusuf What do you think of this?

Reply:
Short answer: no it can't, this is fine.

Long answer: we are incomplete either way, and I doubt that it would help. I am also unsure whether it would bring us any 'closer' to completeness.